code/modules/mob/living/silicon/pai/software.dm

```diff
@@ -158,32 +151,14 @@
 
 			if("image") // Set pAI card display face
 				var/newImage = input("Select your new display image.", "Display Image", "Happy") in sortList(list("Happy", "Cat", "Extremely Happy", "Face", "Laugh", "Off", "Sad", "Angry", "What", "Sunglasses"))
-				var/pID = 1
-
 				switch(newImage)
```

Comment:
If this only checks `null` why not just make this an if?

Reply:
it doesn't only check null.